src/Sql/Sql/help/Restore-AzSqlInstanceDatabase.md

```diff
@@ -403,6 +467,18 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+```yaml
```

Comment:
The heading for this parameter is missing

Reply:
It is not missing, but some parameters are now required in curtain situations. In this case, subscriptionId is required for 2 parameter set, and for others are not. 
Also, I was using given documentation to re-generate md file.